scania-digital-design-system/sdds
# fix: renaming and cleaning demo folder

**Describe pull-request**  

1. Renaming demo to "sandbox" (+ updating all internal paths... )
2. Adding readme file with an explanation of what is a sandbox and why we use it + link to the proper demo page
3. Leaving only Angular version 13 for testing and removing others
4. Adding http-server as a dependency for HTML example

**Solving issue**  
Fixes: AB#1955

**How to test**  
1. Take branch on your local
5. Try to run each environment (HTML/Angular/React/Vue)
6. Check if the sandbox readme file looks OK

Note: @alexlioliopoulos - I do not want to bother with testing the dev environment, just see if sandbox -> readme.md sounds OK to you.
